application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsTranslations.xml

```diff
@@ -261,6 +261,27 @@ rendering.macro.country.endDate=End date
 rendering.macro.country.name=Country
 rendering.macro.country.startDate=Start date
 
+## VisitsOverview Macro
+analytics.visitsOverview.averageVisitDuration.option=Average visit duration
+analytics.visitsOverview.bounced.option=Bounce count
+analytics.visitsOverview.maxActionsInOneVisit.option=Max actions in one visit
+analytics.visitsOverview.nb_actions.option=Number of actions
+analytics.visitsOverview.options.description=The field that is displayed inside the graph.
+analytics.visitsOverview.parameter.label.endDate=End time of the date interval used for this macro
+analytics.visitsOverview.parameter.label.startDate=Start time of the date interval used for this macro
+analytics.visitsOverview.period.day.option=Day
+analytics.visitsOverview.period.description=The period you requested the statistics for. Can be any of: day, week, month, year.
+analytics.visitsOverview.period.month.option=Month
+analytics.visitsOverview.period.week.option=Week
+analytics.visitsOverview.period.year.option=Year
+analytics.visitsOverview.viewsAndUnique.option=Views &amp; Unique views
+analytics.visitsOverview.visits.option=Visits
+rendering.macro.visitsOverview.description=This is an overview of the visit evolution.
+rendering.macro.visitsOverview.name=Visits overview
+
+## Graph errors
+analytics.graph.error=Something went wrong while calling the endpoint.
```

Comment:
The key is a little too generic, it is meant to be used for multiple macros?
For the message: I feel that `endpoint` might be too technical. Maybe `Something went wrong while getting data for the chart.`. WDYT?

Reply:
Yes, the error is meant to be used with multiple macros and indeed the message that you've proposed is better and more fitted for a normal user.